### DISCUSSION TITLE
Is it possible to override supabase Auth with firebase?

### DISCUSSION THREAD

**Discussion by Prashant4900** [supabase/supabase#3115] [General]
This means every user who is created in firebase is automatically added in supabase users. So we can use both of them in one project.

**Comment by silentworks** [comment]
How would you authenticate these users in Supabase? Because firebase and Supabase used different auth mechanism so it would be difficult to keep these in sync.

**Reply by Prashant4900** [reply]
I plan something like that whenever a new user is created in firebase it automatically fetches user details and created a new user in supabase with that information.
